### ISSUE TITLE
  Expression patterns are not showing o...

### ISSUE THREAD

**Issue by wormbasehelp** [WormBase/website#3663]
<p>&nbsp;&nbsp;Expression patterns are not showing on anatomy term pages.  There is a tag for Expression pattern on the Associations widget on the anatomy page, but there are no data showing. <br/>The only way to get anatomy associated expression patterns seems to be through searching for an anatomy term and using the expression pattern class. </p>

Reported by: Kare****** (kare**************)  (obscured for privacy))
Submitted From: AIZ (anatomy_term) <a href="http://www.wormbase.org//species/all/anatomy_term/WBbt:0005836#014--10">/species/all/anatomy_term/WBbt:0005836#014--10</a>

Mozilla/5.0 (Macintosh; Intel Mac OS X 10.10; rv:36.0) Gecko/20100101 Firefox/36.0

**Comment by tharris** [comment]
You sure? AIZ has no expression patterns associated with it in WS246. Other Anatomy terms do. Check for example, DVB.

http://www.wormbase.org/species/all/anatomy_term/WBbt:0004822#01--10

**Comment by kyook** [comment]
There are expression patterns that include AIZ that can be retrieved
through individual gene pages and through searching for AIZ within the
expression pattern class.
http://www.wormbase.org/species/all/expr_pattern/Expr261#021--10

I would think there would be cross referencing of this expression pattern
on the AIZ anatomy page, but instead nothing is mentioned about expression
patterns on AIZ since they are actually assigned to child cells AIZL and
AIZR.

I think it would be good to have some way of getting to the AIZL/R
expression patterns from the parent AIZ page.

Karen



On Thu, Mar 5, 2015 at 2:47 PM, Todd Harris <notifications@github.com>
wrote:

> You sure? AIZ has no expression patterns associated with it in WS246.
> Other Anatomy terms do. Check for example, DVB.
>
> http://www.wormbase.org/species/all/anatomy_term/WBbt:0004822#01--10
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WormBase/website/issues/3663#issuecomment-77469252>.
>



-- 
~~~~
Karen Yook

Curator
WormBase Caltech
Tel: 626.395.5878
e-mail: kyook@caltech.edu
e-mail: kyook@wormbase.org
skype name: wbkaren

**Comment by draciti** [comment]
I agree with Karen. It would be good to see that there are AIZL/R patterns from the parent AIZ page, if possible.

In the Wobr tree is clearer (0 direct annotations) 
http://www.wormbase.org/species/all/anatomy_term/WBbt:0005836#03--10

**Comment by tharris** [comment]
So you want to see expression patterns in child descendants? Immediate descendants only?  Are the expression patterns in AIZR/L not expressed in AIZ?

**Comment by draciti** [comment]
I don't know if it's hard-or possible- to have this way but if we could add in the overview box a 'suggestion' with immediate descendants it would be helpful-see screenshot

![screen shot 2015-03-05 at 4 17 31 pm](https://cloud.githubusercontent.com/assets/901243/6517781/4701f826-c353-11e4-831f-1b55d48358e4.png)

we curate to the most granular term so if is a neuron pair and the authors state AIZ the annotation is made to AIZR and AIZL, not to AIZ.

**Comment by tharris** [comment]
Gotcha. 

I'd like to move away from the simple lists that we present to have something that reads like more of a self-describing narrative. 

It would say something like :

AIZ gives rise to two descendant cells: AIZR and AIZL. 

Expression patterns in AIZ:
Blah 

Expression patterns in descendants:
AIZR blah
Etc

The rationale for this is that we overwhelm users with lists of items. Those are great for displaying a wealth of information but not so good for helping users to understand significance. 

I'm working on something like this for the gene overview right now; this might also be a good use case.

**Comment by draciti** [comment]
Cool. thanks todd

**Comment by raymond91125** [comment]
as Daniela pointed out, expression annotation in descendants (not just children) are indicated already in the Ontology Browser widget. IMHO the tree view is the clearer way to show this hierarchical information in context than that with texts. i suggest that in the Association widget we indicate that there are expression data annotated in descendants and provide a link to open the OB widget.
propagating annotations up or down the hierarchy as default action is not always sensible albeit sometimes bloody useful.

**Comment by draciti** [comment]
Raymond has a point. 
Todd, do you think is feasible to add a sentence in the Association widget saying something like

See expression in descendants of this term by clicking here

and the click will bring the user to the ontology browser widget?

**Comment by draciti** [comment]
g/species/all/anatomy_term/WBbt:0005836#031--10
